packages/web/components/table/asset-info.tsx

```diff
@@ -97,6 +97,15 @@ export const AssetsInfoTable: FunctionComponent<{
     [selectedCategory]
   );
 
+  useMount(() => {
```

Comment:
I think we have nuqs installed, maybe we can use that instead on this? That would be good so we remove location search params sync logic and useState ecc.